### DISCUSSION TITLE
Does SeisSol support the output of surfaces other than free surface? For example, absorbing surface.

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#1227] [Q&A]
![image](https://github.com/user-attachments/assets/31c4623d-16b6-4155-bc6b-757c9595264e)

**Comment by NicoSchlw** [comment]
No, SeisSol does not directly. However, you can choose `OutputRegionBounds` for the 3D wavefield output to check out a certain region without the output size becoming too huge. 
See: https://seissol.readthedocs.io/en/latest/parameter-file.html
